--- libftdi1-1.2/python/CMakeLists.txt.orig	2014-11-21 17:42:17.000000000 +0100
+++ libftdi1-1.2/python/CMakeLists.txt	2015-10-04 08:43:11.144485010 +0200
@@ -12,8 +12,8 @@
       set ( SWIG_FOUND TRUE )
     endif ()
   endif ()
-  find_package ( PythonLibs )
   find_package ( PythonInterp )
+  find_package ( PythonLibs )
 endif ()
 
 if ( SWIG_FOUND AND PYTHONLIBS_FOUND AND PYTHONINTERP_FOUND )
